test/unit/test_data_utils.py

```diff
@@ -31,9 +31,14 @@ def test_get_content_type(self):
 
         self.assertEqual('csv', data_utils.get_content_type('csv'))
         self.assertEqual('csv', data_utils.get_content_type('text/csv'))
+        self.assertEqual('csv', data_utils.get_content_type('text/csv; label_size=1'))
+        self.assertEqual('csv', data_utils.get_content_type('text/csv;label_size = 1'))
+        self.assertEqual('csv', data_utils.get_content_type('text/csv; charset=utf-8'))
 
         with self.assertRaises(exc.UserError):
             data_utils.get_content_type('incorrect_format')
+        with self.assertRaises(exc.UserError):
+            data_utils.get_content_type('text/csv; label_size=5')
```

Comment:
how about `text/csv; label_size`, `text/csv; label_size=1=1` and `text/csv; label_size=1; label_size=2`?

Reply:
Good point, I wasn't thinking of these kinds of edge cases.